0004-bankless-leader.md

```diff
@@ -0,0 +1,126 @@
+---
+simd: '0004'
```

Comment:
The SIMD number needs to be updated to 0005 as per the SIMD-0001 spec for [numbering the SIMDs](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md#draft)

Reply:
Please also update the filename to `0005-bankless-leader.md`?